lib/tasks/handles/fulcrum_published_dois.rake

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# For HELIO-2488
+# HELIO-3118
+
+# Creates a report of handles => dois that will
+# be ingested into TMM so that TMM can know the
+# handle that Fulcrum has assigned to a Monograph
+
+desc 'monograph handles and DOIs for certain presses'
+namespace :heliotrope do
+  task fulcrum_published_dois: :environment do
+    # You can put child presses here, or not.
+    # If you don't it will automatically pick up the children of a press.
+    subdomains = [
+      'michigan',
+      'leverpress',
+      'amherst',
+      'maizebooks',
+      'a2ru',
+      'atg',
+      'cv',
+      'cb',
+      'dialogue'
+    ]
+
+    children = []
+    subdomains.each do |subdomain|
+      children << Press.where(subdomain: subdomain).first&.children&.pluck(:subdomain)
+    end
+
+    subdomains = subdomains + children
+    subdomains = subdomains.flatten.compact.uniq
+
+    hits = ActiveFedora::SolrService.query("{!terms f=press_sim}#{subdomains.map(&:downcase).join(',')}", rows: 100_000)
+
+    hits.each do |hit|
+      next unless hit["suppressed_bsi"] == false && hit["visibility_ssi"] == "open"
```

Comment:
Not really a nitpick but I'm always leery of this `suppressed_bsi` thing. I'm pretty sure action is only ever taken on this in Hyrax/stack code when it's true (Thus removed from results). I have sort of a vague memory of seeing it missing from Solr docs in the past, but I may have made this up. Is that possible?
Either way, I'd like if this read `hit["suppressed_bsi"].blank?` but would welcome correction or discussion.

Reply:
Sounds good to me.  Should added it to rubocop.

Reply:
Also. I would prefer /open/i.match?(hit["visibility_ssi"]), love those case insensitive regular expressions.

Reply:
![thinkin-face](https://user-images.githubusercontent.com/9595080/74460341-bcd01c80-4e5a-11ea-8a78-559af4aa7ce0.png)